cumulusci/tasks/apex/batch.py

```diff
@@ -33,9 +32,28 @@ def _run_task(self):
 
         self.logger.info("Job is complete.")
 
+        vals = {
+            key: value
+            for key, value in self.batch.items()
+            if key
+            in [
+                "Id",
+                "Status",
+                "ExtendedStatus",
+                "NumberOfErrors",
+                "JobItemsProcessed",
+                "TotalJobItems",
+            ]
```

Comment:
Not really consequential, but probably makes more sense to use a set literal here?

Reply:
f0fa6cc2775f05b282dd44d730538e39219b11f6